requirements.txt

```diff
@@ -14,7 +14,7 @@ passlib==1.6.2
 python-dateutil==2.4.2
 python-mimeparse==0.1.4
 six==1.10.0
-git+git://github.com/xcompass/Flask-CAS.git
+git+git://github.com/cameronbwhite/Flask-CAS.git@v1.0.1
```

Comment:
Is the 1.0.1 version support CAS 2 and 3? This PR https://github.com/cameronbwhite/Flask-CAS/pull/10 is merged into a different branch. And has not merged into master. Also it may be missing this commit as well https://github.com/xcompass/Flask-CAS/commit/fcf2093004778ad789712703f2dd4447164f7ebe